specs/0119-transactional-emails.md

```diff
@@ -0,0 +1,428 @@
+
+We hope this gives you a better understanding of how localization works at Mozilla. 
+
+If you have any questions, reach out any time to the Mozilla localization team on [Matrix chat](https://matrix.to/#/#l10n-community:mozilla.org) or [Discourse](https://discourse.mozilla.org/c/l10n/547).
+
+Thanks,  
+Pontoon Team
+
+#### Frequency
+
+Once
+
+### Onboarding: Week 1 - more localization resources
+
+#### Trigger
+
+When both conditions are met: 
+
+1. At least 1 week after account creation  
+2. 5 or more translation suggestions submitted
+
+### Template
+
+Subject: More resources for contributors  
+Body:  
+Hello,
+
+Thank you so much for contributing translations to Mozilla localization! This is our final introductory email - so from here on out you won’t find us in your inbox (excluding essential account related communications) unless you choose to sign up for the future email updates described below.
+
+Since you’ve had a chance to make some translation suggestions, we’d like to provide you with some additional resources that might be useful as you continue in your journey as a member of our localization community.
+
+**Additional documentation:**  
+For in-depth documentation on Mozilla localization, see our [Documentation for Mozilla localizers](https://mozilla-l10n.github.io/localizer-documentation/index.html) which includes in depth explanations on topics such as:
+
+* [Community roles](https://mozilla-l10n.github.io/localizer-documentation/community/l10n_community_roles.html)  
+* Specific [Mozilla product](https://mozilla-l10n.github.io/localizer-documentation/products/index.html) localization (such as Firefox, Firefox for Android, etc.)  
+* Tools such [Pontoon](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/index.html) and [Fluent](https://mozilla-l10n.github.io/localizer-documentation/tools/fluent/index.html) (the advanced translation format used by Mozilla)
+
+**Pontoon Add-on:**   
+You can use the Pontoon Add-on for [Firefox](https://mozilla-l10n.github.io/localizer-documentation/index.html) or [Chrome](https://chromewebstore.google.com/detail/pontoon-add-on/gnbfbnpjncpghhjmmhklfhcglbopagbb) to be instantly notified of any Pontoon notifications right in your browser, as well as some helpful tools to help with proofreading, bug reporting, and fixing. 
+
+**Mozilla community participation guidelines:**  
+As an active member of our community, we’d like you to be aware of the [Mozilla community participation guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) to ensure the wellbeing of all members of the community.
+
+**Email settings:**  
+Finally, if you’re interested in hearing about upcoming events organized for our localization community, be sure to enable email communications in your [settings](https://pontoon.mozilla.org/settings/) so you can be the first to get notified.  
```

Comment:
```suggestion
Finally, if you’re interested in latest updates, announcements about new Pontoon features, invitations to contributor events and more, be sure to enable email communications in your [settings](https://pontoon.mozilla.org/settings/) so you can be the first to get notified.
```

Since only people that contribute in the first N days will get this email, should we move this section to one of the previous intro emails?

Reply:
My intention is to send this to anybody who makes 5 or more suggestions, but sent 1 week or later after account creation (whenever a user reaches 5 suggestions). With that basis, I'd like to leave this in the final email since the user has a higher chance of being serious about contributing versus someone who just signed up.

I think that's a good callout that I should match the legal approved messaging around the opt-in.